### PR TITLE
lintcode: 248-Count-of-Smaller-Number

### PR DESCRIPTION
**Pull request by wen-wayne-shih**
- lintcode: 248-Count-of-Smaller-Number

### CODE REVIEW

File: lintcode/binary_search/248-Count-of-Smaller-Number/solution.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        A.sort()
+        results = []
+        for target in queries:
+            count = self._get_count_of_smaller_number(A, target)
+            results.append(count)
+        return results
+
+    def _get_count_of_smaller_number(self, sorted_nums, target):
+        # <Wayne Shih> 15-Nov-2014
+        # find the first number in A >= q
```
Comment (W-Shih) [inline lintcode/binary_search/248-Count-of-Smaller-Number/solution.py:40]:
```suggestion
        # <Wayne Shih> 15-Nov-2014
        # find the first number in A >= target
```

**Review by W-Shih — CHANGES_REQUESTED** [review]
typo

File: lintcode/binary_search/248-Count-of-Smaller-Number/solution.py (hunk 1 above)
Comment (wen-wayne-shih) [reply lintcode/binary_search/248-Count-of-Smaller-Number/solution.py]: resolved as per suggestion

**Review by W-Shih — APPROVED** [review]
good!

**Pull request merged**